Amplify/DevMenu/Trigger/LongPressGestureRecognizer.swift

```diff
@@ -38,11 +43,20 @@ class LongPressGestureRecognizer: NSObject, TriggerRecognizer, UIGestureRecogniz
     /// Register a `UILongPressGestureRecognizer` to `uiWindow`
     /// to listen to long press events
     @available(iOS 13.0.0, *)
-    private func registerLongPressRecognizer(window: UIWindow) {
-        let longPressRecognizer = UILongPressGestureRecognizer(
-                                    target: self,
-                                    action: #selector(LongPressGestureRecognizer.longPressed(sender:)))
-        longPressRecognizer.delegate = self
-        window.addGestureRecognizer(longPressRecognizer)
+    private func registerLongPressRecognizer() {
+        if longPressGestureRecognizer != nil && uiWindow != nil {
+            longPressGestureRecognizer!.delegate = self
+            uiWindow!.addGestureRecognizer(longPressGestureRecognizer!)
+        }
+    }
+
+    /// Unregisters the long press recognizer from `UIWindow`
+    func destroy() {
```

Comment:
Do we need all of these changes? Recognizer is still attached if we make `LongPressGestureRecognizer` to nil inside `AmplifyDevMenu`.